### ISSUE TITLE
Sticky toolbar

### ISSUE THREAD

**Issue by xu-chris** [JonasDoebertin/kirby-visual-markdown#47]
Hi,

I've played a bit around with your awesome plugin and figured out how you could add a sticky header to it. 
![bildschirmfoto 2015-06-28 um 22 55 04](https://cloud.githubusercontent.com/assets/980778/8398298/e2e21006-1de8-11e5-9355-a9b6f441eda2.png)

It's really rudimentary but this is my solution:

Add this to the init method of your markdown field: 

```
    var  className      = 'fixed',
         $navigation    = $('.visualmarkdown-toolbar'),
         $content       = $('.markdownfield-wrapper');

    $(window).scroll(function() {

        if ($(document).scrollTop() >= $content.offset().top) {
            $navigation.addClass(className);
            $content.css({
                'padding-top': $navigation.height()
            });
            $navigation.css({
                'max-width' : ($content.width())
            })
        }
        else {
            $navigation.removeClass(className);
            $content.css({'padding-top': 0});
        }
    });
```

Add this to your visualmarkdown CSS:

```
.markdownfield-wrapper .visualmarkdown-toolbar.fixed {
    position: fixed;
    top: 0;
    z-index: 999;
}
```

I haven't tried it with multiple markdown fields. But maybe this is helpful for you :-)

**Comment by JonasDoebertin** [comment]
A sticky header is definitely something I want to add soon!

**Comment by JonasDoebertin** [comment]
Implemented with 4f02c5690fe41cc09dc32ec9001512ce862a9c5d and available in the **develop** branch. Needs some in-depth testing though.
